templates/admin/user/list.html

```diff
@@ -14,8 +17,11 @@ <h1 class="title">Users</h1><a class="create" href="/admin/user/create">Create n
     </thead>
     {% for user in users %}
     <tr>
+        <td><img src="data:image/jpg;{{user.avatar}}"></td>
         <td>{{user.gender.value}}</td>
+	<td>{{user.firstName}}</td>
```

Comment:
What about the indentation here, is it changed in the Github representation only?